cockroachcloud/authorization.md

```diff
@@ -23,20 +23,20 @@ See:
 Users may be granted two levels of access:
 
 - 'Developer' allows users to:
-  - Read cluster information in the CockroachDB Cloud Console and Cloud API
+  - Read cluster information using the CockroachDB Cloud Console and Cloud API
 - 'Administrator' allows users to:
   - Create and delete organization users
-  - Create, modify and delete clusters
+  - Create, modify, and delete clusters
   - Create and manage SQL roles on particular clusters
 
-Within a CockroachDB Cloud Organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed over the cluster.
+Within an organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed across the cluster.
```

Comment:
This seems like a bit of a more-complicated repeat of the info that is in line 8. WDYT?

Reply:
this stuff is changing in the FGAC PR, so not trying to over-optimize here. I agree your line is clearer for what it's worth.